core/controllers/admin_test.py

```diff
@@ -476,11 +472,11 @@ def test_upload_topic_similarities(self):
     def test_get_handler_includes_all_feature_flags(self):
         self.login(self.CURRICULUM_ADMIN_EMAIL, is_super_admin=True)
         feature = platform_parameter_registry.Registry.create_feature_flag(
-            PARAM_NAMES.test_feature_1, 'feature for test.', FeatureStages.DEV)
+            ParamNames.TEST_FEATURE_1, 'feature for test.', FeatureStages.DEV)
 
         feature_list_ctx = self.swap(
             platform_feature_services, 'ALL_FEATURES_LIST',
-            [getattr(PARAM_NAMES, feature.name)])
+            [getattr(ParamNames, ParamNames.TEST_FEATURE_1.name)])
```

Comment:
```suggestion
            [ParamNames.TEST_FEATURE_1.value])
```

Reply:
ditto below

Reply:
@vojtechjelinek Sir, When I am using `ParamNames.TEST_FEATURE_1.value` I am getting this error
![paramNames](https://user-images.githubusercontent.com/78922463/163157511-0c70275f-74f0-4d8f-bdd1-77445a1c37f3.png)
So, I am using `ParamNames.TEST_FEATURE_1` as it is getting string value directly without using ".value"